django/contrib/admin/static/admin/js/admin/DateTimeShortcuts.js

```diff
@@ -95,13 +94,11 @@
             }
             message = interpolate(message, [timezoneOffset]);
 
-            var $warning = $('<span>');
-            $warning.attr('class', warningClass);
-            $warning.text(message);
-
-            $(inp).parent()
-                .append($('<br>'))
-                .append($warning);
+            var warning = document.createElement("span");
+            warning.className = warningClass;
+            warning.appendChild(document.createTextNode(message));
```

Comment:
You might want to have a look at the following solutions here:
https://developer.mozilla.org/en-US/docs/Web/API/Node/textContent
https://developer.mozilla.org/en-US/docs/Web/API/Node/innerText
https://developer.mozilla.org/en-US/docs/Web/API/Element/innerHTML

Reply:
And the differences between each of those:
https://developer.mozilla.org/en-US/docs/Web/API/Node/textContent#Description